transports/jms/src/main/java/org/mule/transport/jms/activemq/ActiveMQJmsConnector.java

```diff
@@ -53,6 +53,15 @@ protected void applyVendorSpecificConnectionFactoryProperties(ConnectionFactory
     {
         try
         {
+            Method getPrefetchPolicy = connectionFactory.getClass().getMethod("getPrefetchPolicy");
```

Comment:
Actually, what happens is that activemq dependency is available only for testing. ActiveMQ has to be added by the user to their apps but is not included in the mule distribution.